src/Cuid.php

```diff
@@ -231,4 +231,20 @@ public static function slug()
             $fingerprint .
             $random;
     }
+
+	/**
+	 * Check if string is a valid 'cuid'.
+	 *
+	 * @param cuid: string to check if it is a 'cuid'.
+	 * @return boolean
+	 */
+
+	public static function isCuid($cuid)
+	{
+		if (gettype($cuid) !== 'string') {
+			return false;
+		}
+
+		return $cuid[0] === 'c';
```

Comment:
@aat2703 Do you think you can use RegExp to check the format for the cuid before returning the check result. The cuid itself has a deterministic format.

Reply:
Maybe... it's implemented as in the node library for cuid
source: https://github.com/ericelliott/cuid/blob/753a47ae1016719b7175721858590ebc75ca8c29/index.js#L69

Reply:
Fair enough.